examples/xla_step.py

```diff
@@ -35,14 +39,20 @@ def gym_sync_step() -> None:
   def actor_step(iter, loop_var):
     handle0, states = loop_var
     action = policy(states)
-    handle1, (new_states, rew, done, info) = step(handle0, action)
+    if is_legacy_gym:
+      handle1, (new_states, rew, done, info) = step(handle0, action)
+    else:
+      handle1, (new_states, rew, term, trunc, info) = step(handle0, action)
```

Comment:
have you tested this for new gym version?

Reply:
Yes. It works with both gym=0.26 and gym=0.21.s